.github/workflows/e2e_on_pull.yaml

```diff
@@ -156,7 +156,9 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - centos
+          # FIXME: temporarily disable CentOS tests as the CentOS podvm builds
```

Comment:
I know it's a nit-pick, but there are still 2 indentation ways in this PR. The one used here fits me better but anything that is consistent would be appreciated.